Configuration/StandardSequences/python/Eras.py

```diff
@@ -10,4 +10,25 @@ def __init__(self):
         self.bunchspacing25ns = cms.Modifier()
         self.bunchspacing50ns = cms.Modifier()
 
+        # These eras should not be set directly by the user.
+        self.run2_common = cms.Modifier()
+        self.run2_25ns_specific = cms.Modifier()
+        self.run2_50ns_specific = cms.Modifier()
+        self.run2_HI_specific = cms.Modifier()
+        self.stage1L1Trigger = cms.Modifier()
+        
+        # These are the eras that the user should specify
+        self.Run2_25ns = cms.ModifierChain( self.run2_common, self.run2_25ns_specific, self.stage1L1Trigger )
+        self.Run2_50ns = cms.ModifierChain( self.run2_common, self.run2_50ns_specific )
+        self.Run2_HI = cms.ModifierChain( self.run2_common, self.run2_HI_specific, self.stage1L1Trigger )
```

Comment:
I hadn't realised there was a difference, other than the menu and `simCaloStage1Digis.FirmwareVersion` (from looking at the [`customiseSimL1EmulatorForPostLS1_HI `](https://github.com/cms-sw/cmssw/blob/CMSSW_7_4_0_pre7/L1Trigger/L1TCommon/python/customsPostLS1.py#L129) function).  The `stage1L1Trigger` era I planned to implement everything in the [`customiseSimL1EmulatorForStage1`](https://github.com/cms-sw/cmssw/blob/CMSSW_7_4_0_pre7/L1Trigger/L1TCommon/python/customsPostLS1.py#L10) function and then the other changes (menu and firmware) I was going to put in `run2_HI_specific`.  Are there other differences?